pkg/virt-handler/vm.go

```diff
@@ -401,6 +421,19 @@ func (d *VMHandlerDispatch) processVmUpdate(vm *v1.VM, shouldDeleteVm bool) (boo
 		return isPending, err
 	}
 
+	vif, err := network.SetupVmNetworkInterface(d.domainManager)
```

Comment:
We might want to consider generating these vifs on demand during the InsertBridgeVIF function as it detects network interfaces in the VM definition.  My thought is it's possible that a VM wants more than one interface.